tools/schema/scoreCardMacroSchema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "object",
+  "$id": "scoreCardMacroSchema",
+  "properties": {
+    "title": {
+      "type": "string",
+      "description": "Title text on top of the card (optional)"
```

Comment:
No need to fix to this PR (unless you really want to), but there is no need to highlight that field is "optional". There is `required`  member that defines that.